src/F2_autoreport.py

```diff
@@ -50,6 +52,7 @@
     START_DATE,
     TIMESTEP,
     TIMESERIES_PEAK,
+    TOTAL_FLOW,
```

Comment:
This fixes #357. Please add to PR head!
@mahendrark : You dont need to push this anymore :)